### PR TITLE
Customisation of abbreviations text size upto 14sp

### PR DESCRIPTION
**Pull request by sahanaprabhakar-halodoc**
Abbreviations text size can now be customised. This is tested in the smallest hdpi phone.

### CODE REVIEW

**Comment by SahanaHalodoc** [comment]
@kormateusz Please review.

File: library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt [hunk 1]
```diff
@@ -25,6 +25,15 @@ internal fun View.setAbbreviationsLabels(color: Int, firstDayOfWeek: Int) {
     }
 }
 
+internal fun View.setAbbreviationsLabelsSize(size: Float){
```
Comment (kormateusz) [inline library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt:28]: Please format this class.

File: library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt [hunk 2]
```diff
@@ -25,6 +25,15 @@ internal fun View.setAbbreviationsLabels(color: Int, firstDayOfWeek: Int) {
...
+    val labels = getAbbreviationsTextViews()
+    labels.forEachIndexed{ _, label->
+        if(size>0.0 && size<=28.0) {
```
Comment (kormateusz) [inline library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt:31]: Why below 28.0?

**Review by kormateusz — CHANGES_REQUESTED** [review]

File: library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt (hunk 1 above)
Comment (SahanaHalodoc) [reply library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt]: This is done @kormateusz

File: library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt (hunk 2 above)
Comment (SahanaHalodoc) [reply library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt]: This is resolved in the latest commit.

**Comment by sahanaprabhakar-halodoc** [comment]
@kormateusz Can you please review and merge

**Review by kormateusz — APPROVED** [review]

**Comment by sahanaprabhakar-halodoc** [comment]
@kormateusz can this be merged please

**Comment by sahanaprabhakar-halodoc** [comment]
@kormateusz Any update on when this can be merged?

**Comment by sahanaprabhakar-halodoc** [comment]
@kormateusz Can we merge this please?

**Pull request merged**